firmware/console/binary/tunerstudio_outputs.h

```diff
@@ -229,14 +229,18 @@ typedef struct {
 	scaled_voltage rawOilPressure;		// 242
 
 	int16_t tuneCrc16; // 244
-    uint8_t sd_status; // 246
-	uint8_t pad;
+
+	uint8_t sd_status; // 246
+
+	int8_t tcuCurrentGear; // 247
```

Comment:
hold on, I've seen this before

rebase on master?

Reply:
I'm not sure what's going on. I thought maybe it was just a cached diff.

Reply:
Hang on while I try to wrap my head around git.